app/services/CheckoutService.scala

```diff
@@ -361,15 +364,16 @@ class CheckoutService(identityService: IdentityService,
       }
     }
 
-    def ensureEmail(contact: Contact, subscription: Subscription[WeeklyPlan]) =
-      if (subscription.readerType != ReaderType.Direct || contact.email.isDefined) {
+    def ensureEmail(contact: Queries.Contact, subscription: Subscription[WeeklyPlan]) = {
+      if (contact.email.isDefined) {
```

Comment:
now we are updating it on the bill to in zuora, this also works for giftors renewing their gifted sub